apps/public-docsite-v9/src/Concepts/ChildWindow.stories.mdx

```diff
@@ -21,7 +21,7 @@ function MyComponent(props) {
   const renderer = React.useMemo(() => createDOMRenderer(targetDocument), [targetDocument]);
 
   return (
-    <RendererProvider renderer={renderer}>
+    <RendererProvider renderer={renderer} targetDocument={targetDocument}>
```

Comment:
We didn't update this before, but `RendererProvider` needs `targetDocument` to properly execute rehydration process.